src/appleseed/renderer/modeling/bsdf/diffusebtdf.cpp

```diff
@@ -164,8 +166,10 @@ namespace
             if (cos_in * cos_on < 0.0f)
             {
                 // Compute the BRDF value.
-                value = values->m_transmittance;
-                value *= values->m_transmittance_multiplier * RcpPi<float>();
+                value.set(0.0f);
```

Comment:
There's a bit of redundant work here; maybe we should only clear the AOVs we're not immediately setting right after?